habitat_baselines/rl/ppo/ppo_trainer.py

```diff
@@ -379,6 +379,15 @@ def _eval_checkpoint(
         self.agent.load_state_dict(ckpt_dict["state_dict"])
         self.actor_critic = self.agent.actor_critic
 
+        # get name of performance metric, e.g. "spl"
+        metric_name = self.config.TASK_CONFIG.TASK.MEASUREMENTS[0]
+        metric_cfg = getattr(self.config.TASK_CONFIG.TASK, metric_name)
+        measure_type = baseline_registry.get_measure(metric_cfg.TYPE)
+        assert measure_type is not None, "invalid measurement type {}".format(
+            metric_cfg.TYPE
+        )
+        self.metric_uuid = measure_type(None, None)._get_uuid()
```

Comment:
@JasonJiazhiZhang what `(None, None)` stands for here?

Reply:
Since we just want the uuid, I created a dummy measure object with `(None, None)` and called its `_get_uuid()`.  Although this isn't the most elegant code, I discussed with abhishek and we decided to do it this way :)